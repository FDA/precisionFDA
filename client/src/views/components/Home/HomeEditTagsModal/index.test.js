import React from 'react'
import { shallow } from 'enzyme'

import { HomeEditTagsModal } from '.'


describe('HomeEditTagsModal test', () => {
  it('should render', () => {
    const component = shallow(<HomeEditTagsModal />)

    expect(component).toMatchSnapshot()
  })
})
