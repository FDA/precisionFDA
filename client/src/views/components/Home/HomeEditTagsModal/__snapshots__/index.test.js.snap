// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomeEditTagsModal test should render 1`] = `
<div
  className="home-edit-tags-modal"
>
  <Modal
    modalFooterContent={
      <React.Fragment>
        <Button>
          Cancel
        </Button>
        <Button
          onClick={[Function]}
          type="primary"
        >
          Update tags
        </Button>
      </React.Fragment>
    }
    subTitle="Tags are public to the community"
    title={
      <React.Fragment>
        Edit tags for 
        <i>
          
        </i>
      </React.Fragment>
    }
  >
    <div>
      <TextField
        label="Tags (comma-separated)"
        name="tags"
        onChange={[Function]}
        placeholder="e.g. foo, bar..."
        type="text"
      />
    </div>
  </Modal>
</div>
`;
